### PR TITLE
[Browserless] New Actions

### PR DESCRIPTION
**Pull request by andrewjschuang**
New scraping actions:

- Scrape URL
- Scrape URL List

### <samp>🤖 Generated by Copilot at 1c49623</samp>

This pull request adds two new action components to the browserless app that allow users to scrape content from web pages using the browserless.io service. The components take URLs and HTML selectors as inputs and return the scraped text or the response object. The pull request also updates the browserless app file with a new dependency and some helper methods, and the pnpm-lock.yaml file with a new package version.

### <samp>🤖 Generated by Copilot at 1c49623</samp>

> _`browserless` scrapes_
> _web content with selectors_
> _autumn harvest time_

## WHY

Auxiliary actions for AI.

### <samp>🤖 Generated by Copilot at 1c49623</samp>

*  Add two new action components to scrape content from URLs using browserless.io ([link](https://github.com/PipedreamHQ/pipedream/pull/6145/files?diff=unified&w=0#diff-874ea83ba01ae1a616e20654640ad261cd2d7e0b4fb6088f299a6c8ebcd11d4eR1-R52), [link](https://github.com/PipedreamHQ/pipedream/pull/6145/files?diff=unified&w=0#diff-263d393e44d749d4adf47eacee5071d1da4adb7569b8408bcfa47e3ca9305196R1-R41))
* Modify the browserless app file to add helper methods for making requests to browserless.io ([link](https://github.com/PipedreamHQ/pipedream/pull/6145/files?diff=unified&w=0#diff-ea204d6d0064512e17c4e1e380880e13daa4aef27ba7b7c9dfb2d6648a5c6f29R1-R2), [link](https://github.com/PipedreamHQ/pipedream/pull/6145/files?diff=unified&w=0#diff-ea204d6d0064512e17c4e1e380880e13daa4aef27ba7b7c9dfb2d6648a5c6f29L6-R44))
* Update the pnpm-lock.yaml file to reflect the new version of `@pipedream/platform` ([link](https://github.com/PipedreamHQ/pipedream/pull/6145/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL515-R515))

### CODE REVIEW

File: components/browserless/actions/scrape-url/scrape-url.mjs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import browserless from "../../browserless.app.mjs";
+
+export default {
+  key: "browserless-scrape-url",
+  name: "Scrape URL",
+  description: "Scrape content from a page. [See the documentation](https://www.browserless.io/docs/scrape).",
```
Comment (dylburger) [inline components/browserless/actions/scrape-url/scrape-url.mjs:6]:
```suggestion
  description: "Scrape content from a page. [See the documentation](https://www.browserless.io/docs/scrape)."
```

Comment (dylburger) [reply components/browserless/actions/scrape-url/scrape-url.mjs:6]: Changed "the" to "a" to follow Google's style guide on indefinite articles.

File: components/browserless/actions/scrape-url/scrape-url.mjs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  version: "0.0.1",
+  type: "action",
+  props: {
+    browserless,
+    url: {
+      type: "string",
+      label: "URL",
+      description: "The URL to scrape content from.",
```
Comment (dylburger) [inline components/browserless/actions/scrape-url/scrape-url.mjs:14]:
```suggestion
      description: "The URL to scrape content from.",
```

File: components/browserless/actions/scrape-url/scrape-url.mjs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    selectors: {
+      type: "string[]",
+      label: "Selectors",
+      description: "The HTML tag selectors to extract.",
```
Comment (dylburger) [inline components/browserless/actions/scrape-url/scrape-url.mjs:19]:
```suggestion
      description: "The HTML tag selectors to extract.",
```

File: components/browserless/actions/scrape-url-list/scrape-url-list.mjs [hunk 4]
```diff
@@ -0,0 +1,52 @@
+import browserless from "../../browserless.app.mjs";
+
+export default {
+  key: "browserless-scrape-url-list",
+  name: "Scrape URL List",
+  description: "Scrape content from a list of pages. [See the documentation](https://www.browserless.io/docs/scrape).",
```
Comment (dylburger) [inline components/browserless/actions/scrape-url-list/scrape-url-list.mjs:6]:
```suggestion
  description: "Scrape content from a list of pages. [See the documentation](https://www.browserless.io/docs/scrape)."
```

Comment (dylburger) [reply components/browserless/actions/scrape-url-list/scrape-url-list.mjs:6]: Changed "the" to "a" to follow Google's style guide on indefinite articles.

File: components/browserless/actions/scrape-url-list/scrape-url-list.mjs [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  version: "0.0.1",
+  type: "action",
+  props: {
+    browserless,
+    urls: {
+      type: "string[]",
+      label: "URLs",
+      description: "The list of URLs to scrape content from.",
```
Comment (dylburger) [inline components/browserless/actions/scrape-url-list/scrape-url-list.mjs:14]:
```suggestion
      description: "The list of URLs to scrape content from.",
```

File: components/browserless/actions/scrape-url-list/scrape-url-list.mjs [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    selectors: {
+      type: "string[]",
+      label: "Selectors",
+      description: "The HTML tag selectors to extract.",
```
Comment (dylburger) [inline components/browserless/actions/scrape-url-list/scrape-url-list.mjs:19]:
```suggestion
      description: "The HTML tag selectors to extract.",
```

Comment (dylburger) [reply components/browserless/actions/scrape-url-list/scrape-url-list.mjs:19]: Changed "the" to "The" to follow Google's style guide on capitalization.

**Review by lcaresia — APPROVED** [review]
LGTM!

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[Browserless_6145_1709.pdf](https://github.com/PipedreamHQ/pipedream/files/11321316/Browserless_6145_1709.pdf)

**Pull request merged**